src/auxinfo/proof.rs

```diff
@@ -12,13 +12,16 @@ use crate::{
     messages::{AuxinfoMessageType, Message, MessageType},
     participant::InnerProtocolParticipant,
     ring_pedersen::VerifiedRingPedersen,
-    zkp::{
-        pifac::{CommonInput, PiFacProof, ProverSecret},
-        pimod::{PiModInput, PiModProof, PiModSecret},
-        Proof, ProofContext,
-    },
     Identifier,
 };
+//zkp::pifac::CommonInput
+use crate::zkp::pifac::PiFacProof;
+//use crate::zkp::pifac::ProverSecret;
+use crate::zkp::{Proof, ProofContext};
+//use crate::zkp::pimod::CommonInput;
+use crate::zkp::pimod::PiModProof;
+//use zkp::pimod::ProverSecret;
```

Comment:
todo: replace this with
```
use crate::zkp::{pifac, pimod, Proof, ProofContext};
```

Then, inline throughout the file, remove any `crate::zkp::` prefixes.